homeassistant/components/http/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "http",
   "name": "HTTP",
+  "after_dependencies": ["isal"],
```

Comment:
After dep to ensure its updated before loading http on core installs

Reply:
does this work? http is loaded as one of the first things and it won't pull in after dependencies

Reply:
You would have to add it to https://github.com/home-assistant/core/blob/dev/homeassistant/bootstrap.py#L143 I think? So it gets promoted.

Reply:
I fixed pre-stage1 after deps in https://github.com/home-assistant/core/pull/114045

Reply:
```
2024-05-05 15:49:29.926 INFO (SyncWorker_6) [homeassistant.util.package] Attempting install of isal==1.6.1
2024-05-05 15:49:33.176 WARNING (MainThread) [homeassistant.helpers.config_validation] The 'base_url' option near /Users/bdraco/home-assistant/config/configuration.yaml:43 is deprecated, please remove it from your configuration
2024-05-05 15:49:33.176 INFO (MainThread) [homeassistant.setup] Setting up http
2024-05-05 15:49:33.176 INFO (MainThread) [homeassistant.setup] About to setup http
2024-05-05 15:49:33.394 INFO (MainThread) [homeassistant.setup] Completed setup http
```